### PR TITLE
Most the triggers should be right matching the new single alias

### PR DESCRIPTION
**Pull request by Ramit110**

### CODE REVIEW

File: scopes.cwt [hunk 1]
```diff
@@ -51,37 +47,27 @@ alias[effect:else] = {
...
-}
+alias[trigger:<society>] = single_alias_right[trigger_clause]
 
 ## scope = any
 ## push_scope = society
 ### Scopes to a society by absolute reference.
-alias[effect:<society>] = {
-    alias_name[effect] = alias_match_left[effect]
-}
+alias[effect:<society>] = single_alias_right[trigger_clause]
```
Comment (Ramit110) [inline scopes.cwt]: Missed it

File: scopes.cwt [hunk 2]
```diff
@@ -51,37 +47,27 @@ alias[effect:else] = {
...
-}
+alias[trigger:<offmap>] = single_alias_right[trigger_clause]
 
 ## scope = any
 ## push_scope = offmap
 ### Scopes to an offmap power by absolute reference.
-alias[effect:<offmap>] = {
-    alias_name[effect] = alias_match_left[effect]
-}
+alias[effect:<offmap>] = single_alias_right[trigger_clause]
```
Comment (Ramit110) [inline scopes.cwt]: And here

File: scopes.cwt [hunk 3]
```diff
@@ -108,37 +92,27 @@ alias[effect:c_<character>] = {
...
-}
+alias[trigger:<trait.cached>] = single_alias_right[trigger_clause]
 
 ## scope = any
 ## push_scope = character
 ### Scopes to all characters with a cached trait.
-alias[effect:<trait.cached>] = {
-    alias_name[effect] = alias_match_left[effect]
-}
+alias[effect:<trait.cached>] = single_alias_right[trigger_clause]
```
Comment (Ramit110) [inline scopes.cwt]: Here too

File: scopes.cwt [hunk 4]
```diff
@@ -108,37 +92,27 @@ alias[effect:c_<character>] = {
...
-}
+alias[trigger:enum[provinces]] = single_alias_right[trigger_clause]
 
 ## scope = any
 ## push_scope = province
 ### Scopes to a province by absolute reference.
-alias[effect:enum[provinces]] = {
-    alias_name[effect] = alias_match_left[effect]
-}
+alias[effect:enum[provinces]] = single_alias_right[trigger_clause]
```
Comment (Ramit110) [inline scopes.cwt]: This as well

**Comment by Ramit110** [comment]
AFAIK, this is done.

**Comment by Ramit110** [comment]
Uuuuh

**Comment by UristMcDorf** [comment]
Looks good to me.

**Pull request merged**